### PR TITLE
Fix for TelegramJsonParser

### PR DESCRIPTION
**Pull request by galatolofederico**
Messages in my Telegram JSON exports occasionally lack the "from" and "text" attributes. This PR fixes that.

### CODE REVIEW

File: chatminer/chatparsers.py [hunk 1]
```diff
@@ -237,12 +237,19 @@ def _read_file_into_list(self):
...
-            "author": mess["from"],
-            "datetime": datetime.datetime.fromtimestamp(int(mess["date_unixtime"])),
-            "message": mess["text"],
-        }
-        return parsed_message
+        if (
+            "from" in mess
+            and "text" in mess
+            and type(mess["text"]) is str
+            and len(mess["text"]) > 0
```
Comment (joweich) [inline chatminer/chatparsers.py:244]: @galatolofederico Do we need these two additional checks? In which cases would they return `false`, while the first two checks return `true`?
If we don't need them, I would drop them for codestyle and performance reasons.

**Review by joweich — COMMENTED** [review]
Left one small comment. Thank you for your contribution @galatolofederico!

**Comment by galatolofederico** [comment]
Hi honestly `type(mess["text"]) is str` is a bit of a workaround because it filters only the pure text messages.
`text` can be either a string if the message contains only text or a list containing strings and dicts.

For example

```
    {
      "id": ...,
      "type": "message",
      "date": "...",
      "date_unixtime": "...",
      "from": "...",
      "from_id": "...",
      "text": [
        "This is some text followed by ",
        {
          "type": "link",
          "text": "https://a-link"
        }
      ],
      "text_entities": [
        {
          "type": "plain",
          "text": "This is some text followed by "
        },
        {
          "type": "link",
          "text": "https://a-link"
        }
      ]
    },
```

The second check `len(mess["text"]) > 0` is redundant i guess.
Maybe the best approach would be to use `text` as is, if it is a string otherwise rebuild the original message from the data structure.
Let me know what you think and i will update the PR

**Comment by joweich** [comment]
Agree with your proposal of reconstructing the original message in case the text field is not a native string.
Really happy that you're contributing to this project!🤗

**Comment by galatolofederico** [comment]
Thank you for sharing this amazing project!

**Comment by joweich** [comment]
You might have a look at #18 if you want to leverage your expertise in telegram exports even more!

**Pull request merged**